.github/pull_request_template.md

```diff
@@ -0,0 +1,61 @@
+### Description
+
+<!--
+A description of the change in this PR.
+Place a reference to the Github issue or the PagerDuty incident, if any, here.
+-->
+
+### Type of Change
+
+- [X] Routine: low risk change
+- [ ] Declared: a significant or high risk change
+- [ ] Emergency: a change made to fix an on-going incident
+
+<!--
+Classify this change as one of the following, according to the change management
+policy in https://drive.google.com/drive/folders/1YgEceW5HQ2bQ4OAuhFIgN1oHmLk9RfTV
+-->
+
+### Testing
+
+- [ ] None: Implicit through tooling or non-configuration management change
+- [ ] Unit Tests
+- [ ] Functional Tests
+- [X] Manual Tests
+
+<!--
+Describe the testing procedure that has been done, or will be done, if any.
+Optional for routine changes, mandatory for declared changes.
+-->
+
+### Rollout Procedure
+
+- [ ] None: Non-configuration management change
+- [ ] Manual Roll-Out: <!-- If checked, replace with description -->
+- [X] Automated Roll-Out (Full): Changes will be applied when an image is built from master and pushed to redcanary/cb-event-forwarder:latest
```

Comment:
So, docker image builds are not part of the change roll out, but expected to be part of the development process itself, specifically with the build portion. The roll-out, in this case, would be the updating of the event-forwarder image within kubernetes, which would in turn update the containers. Im not super up to speed with how updated images swap out with current running images, but that is what needs to be explained here, and in an SOP for changes to this repo.

Reply:
I don't know as we have a procedure for this to be honest. I checked and circle ci is configured to build and push to Dockerhub when a change is committed to master.  I think roll-outs would always be considered manual though currently.  

 I think they're really would be two roll-outs if the code for the forwarder itself was updated.  I'm basing this on the assumption that this is both the forwarder we run on CB servers and the one we run in Docker on Kubernernetes. 

With the current state we'd have to manually roll each deployment so a new pod spawned, also with the hpa we could potentially be running two different versions i as the newly scaled up pods will probably pull latest unless they're image pull is such they use the one already on the node. 


Checkbox updated to default to manual.